content/Video_Display.md

```diff
@@ -620,7 +620,8 @@ Y=160 hides a sprite.
 
 ### Byte1 - X Position
 
-X = Sprite's horizontal position on the screen + 8. An
+X = Sprite's horizontal position on the screen + 8. This works like the examples
```

Comment:
"This works like the examples above but simpler" doesn't sound clear enough for me.
Maybe "This works similarly to the examples above"